ninja-core/src/main/java/ninja/template/TemplateEngineFreemarkerExceptionHandler.java

```diff
@@ -53,38 +52,9 @@ public TemplateEngineFreemarkerExceptionHandler(Logger logger,
 
     public void handleTemplateException(TemplateException te,
                                         Environment env,
-                                        Writer out) {
+                                        Writer out) throws TemplateException {
 
-        if (ninjaProperties.isProd()) {
-            
-            PrintWriter pw = (out instanceof PrintWriter) ? (PrintWriter) out
-                    : new PrintWriter(out);
-            pw.println(
-                     "<script language=javascript>//\"></script>"
-                    + "<script language=javascript>//\'></script>"
-                    + "<script language=javascript>//\"></script>"
-                    + "<script language=javascript>//\'></script>"
-                    + "</title></xmp></script></noscript></style></object>"
-                    + "</head></pre></table>"
-                    + "</form></table></table></table></a></u></i></b>"
-                    + "<div align=left "
-                    + "style='background-color:#FFFF00; color:#FF0000; "
-                    + "display:block; border-top:double; padding:2pt; "
-                    + "font-size:medium; font-family:Arial,sans-serif; "
-                    + "font-style: normal; font-variant: normal; "
-                    + "font-weight: normal; text-decoration: none; "
-                    + "text-transform: none'>");
-            pw.println("<b style='font-size:medium'>Ooops. A really strange error occurred. Please contact admin if error persists.</b>");
-            pw.println("</div></html>");
-            pw.flush();
-            pw.close();
-            
-            logger.log(Level.SEVERE, "Templating error. This should not happen in production", te);
-            
-            
-
-        } else {
-            
+        if (!ninjaProperties.isProd()) {
```

Comment:
Woops! You're correct, the generic error page eats the template error page when we always re-throw.